driver/network_driver.go

```diff
@@ -286,12 +286,12 @@ func (d NetworkDriver) CreateEndpoint(request *network.CreateEndpointRequest) (*
 		// the request if the profile already exists.
 		profile := &api.Profile{
 			Metadata: api.ProfileMetadata{
-				Name: networkData.Name,
-				Tags: []string{networkData.Name},
```

Comment:
Strictly speaking, a Policy can reference Tags in its rules, so this is a breaking change.

It may be that we don't care, since Tags have been deprecated for long enough that people shouldn't be using them, but good to be aware of.